### DISCUSSION TITLE
umi4 document.ejs 文件没有了吗

### DISCUSSION THREAD

**Discussion by jxh150535011** [umijs/umi#8444] [Q&A]
RT

**Comment by fz6m** [comment]
See https://github.com/umijs/umi-next/issues/868

**Reply by jrongbin** [reply]
请问，这个路径里的内容还有办法给恢复出来吗？现在打开是404了。我原本打算给 document.ejs 的 <body> 上变成一个 <body id="html-body">,现在这样就加不了了。添加这个属性是为了我更加方便的使用 css 去覆盖部分 css，特别是 modal 的样式，在 modal 弹出的那一刻，实际是将 modal 的 dom 建立在 body 下，并不在 
`<div id="root">`
里面。
